### ISSUE TITLE
Disk activity plot showing nothing, all drives

### ISSUE THREAD

**Issue by alex-powell** [rockstor/rockstor-core#2049]
Disk activity plot showing nothing, all drives

Steps to replicate:
Go onto Rockstor home page
“Disk Activity” shows nothing

```
[root@rockstor ~]# cat /proc/diskstats
8 0 sda 2103966 47379 2887542896 224503998 6754359 3074361 3503648076 2714493765 0 88186837 2938986300
8 1 sda1 220 9 15408 13949 30 8 148 1430 0 10381 15379
8 2 sda2 100 0 10568 5190 0 0 0 0 0 4275 5189
8 3 sda3 2103542 47370 2887510544 224478840 6379430 3074353 3503647928 2683896486 0 64378901 2908350795
11 0 sr0 0 0 0 0 0 0 0 0 0 0 0
179 0 mmcblk0 379 0 25688 2324 0 0 0 0 0 1076 2325
179 1 mmcblk0p1 300 0 19392 1950 0 0 0 0 0 841 1951
8 16 sdb 571091 945518 627227000 35678443 291100 150010 301056720 94595518 0 8136070 130282402
[root@rockstor ~]#

```
```
[root@rockstor ~]# ls -la /dev/disk/by-id
total 0
drwxr-xr-x 2 root root 240 May 28 18:50 .
drwxr-xr-x 6 root root 120 May 25 10:36 …
lrwxrwxrwx 1 root root 9 May 28 18:51 ata-TSSTcorp_DVD+_-RW_SN-208BB_R8KL6GCC500J5C -> …/…/sr0
lrwxrwxrwx 1 root root 9 May 28 18:51 ata-WDC_WD10JPVT-75A1YT0_WX61EC1UFC25 -> …/…/sda
lrwxrwxrwx 1 root root 10 May 28 18:51 ata-WDC_WD10JPVT-75A1YT0_WX61EC1UFC25-part1 -> …/…/sda1
lrwxrwxrwx 1 root root 10 May 28 18:51 ata-WDC_WD10JPVT-75A1YT0_WX61EC1UFC25-part2 -> …/…/sda2
lrwxrwxrwx 1 root root 10 May 28 18:51 ata-WDC_WD10JPVT-75A1YT0_WX61EC1UFC25-part3 -> …/…/sda3
lrwxrwxrwx 1 root root 9 May 28 18:51 usb-WD_My_Passport_0820_57584231453834455A59374D-0:0 -> …/…/sdb
lrwxrwxrwx 1 root root 9 May 28 18:51 wwn-0x50014ee657c222ee -> …/…/sda
lrwxrwxrwx 1 root root 10 May 28 18:51 wwn-0x50014ee657c222ee-part1 -> …/…/sda1
lrwxrwxrwx 1 root root 10 May 28 18:51 wwn-0x50014ee657c222ee-part2 -> …/…/sda2
lrwxrwxrwx 1 root root 10 May 28 18:51 wwn-0x50014ee657c222ee-part3 -> …/…/sda3
[root@rockstor ~]#

```
Checked the browser console:
```
Uncaught TypeError: Cannot read property ‘resize’ of null
at child.resize (storageadmin.js:20667)

storageadmin.js:20667:
_this.TopDisksChart.resize()
```

```
[root@rockstor ~]# yum info rockstor
Loaded plugins: changelog, fastestmirror
Loading mirror speeds from cached hostfile

base: ftp.swin.edu.au
epel: fedora.melbourneitmirror.net
extras: ftp.swin.edu.au
updates: ftp.swin.edu.au
Installed Packages
Name : rockstor
Arch : x86_64
Version : 3.9.1
Release : 16
Size : 85 M
Repo : installed
From repo : Rockstor-Testing
Summary : RockStor – Store Smartly
License : GPL
Description : RockStor – Store Smartly
```
https://forum.rockstor.com/t/disk-activity-plot-showing-nothing-all-drives/6147/4

**Comment by phillxnet** [comment]
@alex-powell Thanks for the detailed report and for linking back to your forum thread.
I don't see your quoted browser console error here on latest Chrome and late Firefox. Can you report the browser details you are experiencing this message on? May not be related but good to know just in case.

Linking for context to later improvements made to this area of code that may be relevant:
"regression in dashboard disk activity widget" #1978
and it's associated pr:
"regression in dashboard disk activity widget. Fixes #1978" #1979
With the latter released in stable channel as 3.9.2-42 (1 Nov 2018)

However as the linked issue/pr set pertains to multiple same length by-id names, which are not seen in this issue, and relate to a June 2018 regression, their relevance may be only the include unit tests which should be extended, if need by, in the process of this issues 'fix'. Assuming this issue can be reproduced in the latest stable channel release code.

**Comment by phillxnet** [comment]
Closing as pertaining to now defunct base OS. We do have a similar Tumbleweed only issue re no-show disk activity widget with our pending 5.0.9-0 installers but that will be covered in a new issue as it is seen only on TW base, Leap bases of 15.4, 15.5, 15.6 are not currently affected.
